### ISSUE TITLE
[BUG] MKS robin nano v.2, MKSUI with Wifi, Compilation error

### ISSUE THREAD

**Issue by Malderin** [MarlinFirmware/Marlin#22082]
### Did you test the latest `bugfix-2.0.x` code?

Yes, and the problem still exists.

### Bug Description

Board: MKS robin nano v.2
Last marlin 2.x bugfix
[Configs.zip](https://github.com/MarlinFirmware/Marlin/files/6618510/Configs.zip)

Compilation error with enabled wifi ooption(#define MKS_WIFI_MODULE):

```
In file included from Marlin/src/lcd/extui/mks_ui/draw_ui.h:85,
                 from Marlin/src/MarlinCore.cpp:72:
Compiling .pio/build/mks_robin_nano35/src/src/feature/tmc_util.cpp.o
Marlin/src/lcd/extui/mks_ui/wifiSerial.h:38:10: fatal error: libmaple/libmaple_types.h: No such file or directory
   38 | #include <libmaple/libmaple_types.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```

**Comment by ellensp** [comment]
I cannot replicate this error.

I get a different error

```
In file included from Marlin/src/lcd/extui/mks_ui/draw_ui.h:85,
                 from Marlin/src/MarlinCore.cpp:72:
Compiling .pio/build/mks_robin_nano35/src/src/feature/tmc_util.cpp.o
Marlin/src/lcd/extui/mks_ui/wifiSerial.h:38:10: fatal error: libmaple/libmaple_types.h: No such file or directory
   38 | #include <libmaple/libmaple_types.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```
this is due to MKS_WIFI_MODULE, which has not been upgraded away from the old maple code. 
Ie it only works under mks_robin_nano35_maple build environment at this time.

**Comment by ellensp** [comment]
You probably need to delete the hidden .pio directory in your marlin folder.
If you built under the older mks_robin_nano35 then upgraded marlin, platformio doesn't notice that mks_robin_nano35 is not the same as it was and joins old and new build environments. 
Also a build clean will not remove enough, you need to delete the .pio folder

**Comment by Malderin** [comment]
> I cannot replicate this error.
> 
> I get a different error
> 
> ```
> In file included from Marlin/src/lcd/extui/mks_ui/draw_ui.h:85,
>                  from Marlin/src/MarlinCore.cpp:72:
> Compiling .pio/build/mks_robin_nano35/src/src/feature/tmc_util.cpp.o
> Marlin/src/lcd/extui/mks_ui/wifiSerial.h:38:10: fatal error: libmaple/libmaple_types.h: No such file or directory
>    38 | #include <libmaple/libmaple_types.h>
>       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
> compilation terminated.
> ```
> 
> this is due to MKS_WIFI_MODULE, which has not been upgraded away from the old maple code.
> Ie it only works under mks_robin_nano35_maple build environment at this time.


If you specify:
#define X_DRIVER_TYPE  TMC2209
#define Y_DRIVER_TYPE  TMC2209
#define Z_DRIVER_TYPE  TMC2209
#define E0_DRIVER_TYPE TMC2209
 you will get an error like I have above.

With a4988 I also get an error:

```
In file included from Marlin\src\lcd/extui/mks_ui/draw_ui.h:85,
                 from Marlin\src\MarlinCore.cpp:72:
Marlin\src\lcd/extui/mks_ui/wifiSerial.h:38:10: fatal error: libmaple/libmaple_types.h: No such file or directory
   38 | #include <libmaple/libmaple_types.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by Malderin** [comment]
> You probably need to delete the hidden .pio directory in your marlin folder.
> If you built under the older mks_robin_nano35 then upgraded marlin, platformio doesn't notice that mks_robin_nano35 is not the same as it was and joins old and new build environments.
> Also a build clean will not remove enough, you need to delete the .pio folder

I deleted the .pio and ..piolibdeps folders, now I get an error:
```
In file included from Marlin/src/lcd/extui/mks_ui/draw_ui.h:85,
                 from Marlin/src/MarlinCore.cpp:72:
Compiling .pio/build/mks_robin_nano35/src/src/feature/tmc_util.cpp.o
Marlin/src/lcd/extui/mks_ui/wifiSerial.h:38:10: fatal error: libmaple/libmaple_types.h: No such file or directory
   38 | #include <libmaple/libmaple_types.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

With the wi-fi option disabled (//  #define MKS_WIFI_MODULE), there are no compilation errors.

**Comment by ellensp** [comment]
Using your provided files, without modification (ie they have #define X_DRIVER_TYPE  TMC2209, #define Y_DRIVER_TYPE  TMC2209, #define Z_DRIVER_TYPE  TMC2209 etc)

I can not reproduce your error

**Comment by Malderin** [comment]
> Using your provided files, without modification (ie they have #define X_DRIVER_TYPE TMC2209, #define Y_DRIVER_TYPE TMC2209, #define Z_DRIVER_TYPE TMC2209 etc)
> 
> I can not reproduce your error

I updated the comment above, you're right.

**Comment by ellensp** [comment]
I believe  this PR will fix this https://github.com/MarlinFirmware/Marlin/pull/22109

Also this is now just a  Duplicate of https://github.com/MarlinFirmware/Marlin/issues/21069

So I'm closing it.

**Comment by Malderin** [comment]
> I believe this PR will fix this #22109

With this PR a new error:

```
Compiling .pio\build\mks_robin_nano35\src\src\lcd\extui\mks_ui\tft_lvgl_configuration.cpp.o
Marlin\src\lcd\extui\mks_ui\tft_lvgl_configuration.cpp:49:12: fatal error: ../../../../module/servo.h: No such file or directory
   49 |   #include "../../../../module/servo.h"
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Compiling .pio\build\mks_robin_nano35\src\src\lcd\extui\mks_ui\tft_multi_language.cpp.o
*** [.pio\build\mks_robin_nano35\src\src\lcd\extui\mks_ui\tft_lvgl_configuration.cpp.o] Error 1
 
========================= [FAILED] Took 250.04 seconds =========================
Environment       Status    Duration
----------------  --------  ------------
mks_robin_nano35  FAILED    00:04:10.038
==================== 1 failed, 0 succeeded in 00:04:10.038 ====================
```

**Comment by Malderin** [comment]
I think the wrong path is indicated. In tft_lvgl_configuration.cpp there is so:
```
#include "../../../../module/servo.h"
#include "../../../../module/probe.h"
```

And it should be like this:
```
#include "../../../module/servo.h"
#include "../../../module/probe.h"
```
